apps/ledger-live-mobile/src/newArch/features/Web3Hub/screens/Web3HubTabs/components/TabItem/index.tsx

```diff
@@ -0,0 +1,105 @@
+import { Flex, IconsLegacy, Text } from "@ledgerhq/native-ui";
+import type { TabsProps } from "LLM/features/Web3Hub/types";
+import React, { useCallback, useState } from "react";
+import { Image, StyleSheet, TouchableOpacity, View } from "react-native";
+import { Theme } from "~/colors";
+import { NavigatorName, ScreenName } from "~/const";
+
+export type Web3HubTabType = {
+  id: string;
+  manifestId: string;
+  icon: string | null | undefined;
+  title: string;
+  previewUri: string;
+  url: string | URL;
+};
+
+export default function TabItem({
+  item,
+  onItemClosePress,
+  extraData,
+  navigation,
+}: {
+  item: Web3HubTabType;
+  onItemClosePress: (itemId: string) => void;
+  navigation: TabsProps["navigation"];
+  extraData: { colors: Theme["colors"] };
+}) {
+  const [imageLoaded, setImageLoaded] = useState(true);
+  const handleImageLoad = useCallback(() => setImageLoaded(true), []);
+  const handleImageError = useCallback(() => setImageLoaded(false), []);
+  const { colors } = extraData;
+
+  const goToApp = useCallback(() => {
+    navigation.push(NavigatorName.Web3Hub, {
+      screen: ScreenName.Web3HubApp,
+      params: {
+        manifestId: item?.manifestId,
+      },
+    });
+  }, [navigation, item?.manifestId]);
+
+  const handleClosePress = () => {
+    onItemClosePress(item.id);
+  };
```

Comment:
We should probably keep using `useCallback` here
```suggestion
  const handleClosePress = useCallback(() => {
    onItemClosePress(item.id);
  }, [item.id, onItemClosePress]);
```